src/views/VendorOverview.vue

```diff
@@ -1,55 +1,125 @@
 <template>
-  <div class="vendor-overview container mx-auto mb-8 p-24 space-y-40 pb-3">
-    <div class="flex flex-col items-center space-y-8">
-      <img class="w-30 inset-12" alt="Augustin logo" :src="apiUrl + 'img/logo.png'" />
-      <h2 class="text-3xl font-bold">Dein QR-Code</h2>
-      <img class="w-30 inset-12" alt="vendor qr code" :src="qrcode" />
-    </div>
-    <br />
-    <div class="my-16 container mt-8 pt-8 space-y-4">
-      <h2 class="text-3xl font-bold">Dein Guthaben</h2>
-      <div class="vendor-credit text-6xl font-bold">{{ credit }}€</div>
-      <div class="vendor-number">Ausweisnummer: {{ idNumber }}</div>
-      <div class="vendor-name">Name: Doris</div>
-      <button class="bg-green-500 rounded-full py-3 px-6 text-white font-bold">Logout</button>
-    </div>
-  </div>
+  <component :is="$route.meta.layout || 'div'">
+    <template #main>
+      <div className="vendor-overview container mb-8 space-y-40 pb-3 w-5/6">
+        <div className="flex flex-col items-center space-y-8">
+          <h2 className="text-2xl font-bold">QR-Code</h2>
+          <div id="canvas"></div>
+          <div><strong>Url:</strong> {{ vendorMe?.UrlID }}</div>
+          <div>
+            <strong>{{ $t('IDNumber') }}</strong
+            >: {{ vendorMe?.LicenseID }}
+          </div>
+        </div>
+        <br />
+        <div class="information my-16 mt-8 pt-8 space-y-4 text-sm">
+          <h2 class="text-2xl font-bold text-center">{{ $t('yourData') }}</h2>
+          <div class="grid grid-cols-2 place-content-between">
+            <strong>Name:</strong>
+            {{ vendorMe?.FirstName }} {{ vendorMe?.LastName }}
+          </div>
+          <div class="grid grid-cols-2 place-content-between">
+            <strong>Email: </strong>{{ vendorMe?.Email }}
+          </div>
+          <div class="grid grid-cols-2 place-content-between">
+            <strong> {{ $t('telephone') }}: </strong>{{ vendorMe?.Telephone }}
+          </div>
+          <div class="grid grid-cols-2 place-content-between">
+            <strong>{{ $t('location') }}: </strong>{{ vendorMe?.Location }}
+          </div>
+          <div class="grid grid-cols-2 place-content-between">
+            <strong>{{ $t('postCode') }}:</strong> {{ vendorMe?.PLZ }}
+          </div>
```

Comment:
We don't want to show telephone and email here

Reply:
I left them (in the new profile page) after internal discussions